pvnet/models/multimodal/multimodal.py

```diff
@@ -253,10 +274,27 @@ def forward(self, x):
 
         # ********************** Embedding of GSP ID ********************
         if self.embedding_dim:
-            id = x[BatchKey.gsp_id][:, 0].int()
+            if self.target_key_name == "wind":
```

Comment:
Would it be possible to clean with up with something like `id = x[BatchKey[f"{self.target_key_name}_id"]][:, 0].int()`? Not so sure about this one though